tests/Fixtures/resources.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<resources>
+    <configdummy>
```

Comment:
I suggest we remove resource id from the XML config because it's useless. It makes the YAML more readable so we should keep that.